server/pulp/server/db/model/migration_tracker.py

```diff
@@ -24,29 +15,12 @@ class MigrationTracker(Model):
     :type version: int
     """
 
-    collection_name = 'migration_trackers'
-    unique_indices = ('name',)
+    name = StringField(unique=True, required=True)
+    version = IntField(default=0)
+    # For backward compatibility
+    _ns = StringField(default='migration_trackers')
 
-    def __init__(self, name, version):
-        """
-        Initialize the MigrationTracker with name and version.
-
-        :param name:    The name is used to store the name of the migration package this object is
-                        tracking
-        :type  name:    str
-        :param version: The version we want to set for the MigrationTracker
-        :type  version: int
-        """
-        super(MigrationTracker, self).__init__()
-
-        self.name = name
-        self.version = version
-
-    def save(self):
-        """
-        Save any changes made to this MigrationTracker to the database. If it doesn't exist in the
-        database already, insert a new record to represent it.
-        """
-        self.get_collection().update({'name': self.name},
-                                     {'name': self.name, 'version': self.version},
-                                     upsert=True, safe=True)
+    meta = {'collection': 'migration_trackers',
+            'indexes': [],  # small collection, does not need an index
+            'allow_inheritance': False,
+            'queryset_class': CriteriaQuerySet}
```

Comment:
Does this actually need `CriteriaQuerySet`? I think all this code will work fine with the generic queryset manager. If that is the case this line should be deleted.